python/lsst/ts/xml/generate_subsystems_doc.py

```diff
@@ -291,12 +291,19 @@ def generate_subsystems_doc():
                             write_heading(cf, name="Enumerations")
                             enumeration_seen = True
                         states = topic.text.split(",")
+                        print(states)
                         state_name = states[0].split("_")[0].lstrip()
                         cf.write(f":{state_name}:\n")
                         for state in states:
+                            state = state.split("_")[1:]
+                            state = "".join(state)
```

Comment:
Please consider replacing the above two lines with:
```
state = state.split("_", 1)[1]
```
where the 2nd argument to `split` specifies the maximum number of splits.